blog/resources/views/layout/main.php

```diff
@@ -137,6 +137,11 @@
                         'url' => $urlGenerator->generate('auth/signup'),
                         'visible' => $isGuest,
                     ],
+                    [
```

Comment:
Usually the link isn't added to main menu but is present in the login form since user enters incorrect password there and searching for reset.

Reply:
Oops yes I am signing up my clients internally with the cookie login interface and I have a fixed number of clients so that there is no need to send a reset url.  So don't need a public url for them on the login form.

Instead they need to know their current password in order to change their password and have to be logged in, in order to change it  => in the main layout. If they forget their password, they consult me directly via. mobile.